docs/_themes/clusterhq/layout_docs.html

```diff
@@ -65,6 +65,8 @@
             })();
         </script>
     {% endif %}
+	
+    <link rel="shortcut icon" href="https://clusterhq.com/assets/favicon.ico"/>
```

Comment:
Also, do we have a policy about mixed content (http and https)?  Should a URL like this be protocol-relative (`//domain/path`)?

A quick search of the internet suggest people don't like this anymore and everything should just be `https` all the time, so maybe not relevant (except maybe the doc standard should point this out, if this is in fact what we want to do - I note that immediately above we use the ga blob that picks http or https for you to avoid mixed content, maybe we want to fix that).